hubble-fe/src/i18n/resources/zh-CN/graph-managment/addition.json

```diff
@@ -1,15 +1,54 @@
 {
   "addition": {
+    "operate": {
+      "detail": "详情",
+      "favorite": "收藏",
+      "load-statement": "加载语句",
+      "time": "时间",
+      "name": "名称",
+      "favorite-statement": "收藏语句",
+      "operate": "操作",
+      "modify-name": "修改名称",
+      "execution-record": "执行记录",
+      "favorite-queries": "收藏的查询",
+      "favorite-search-desc": "搜索收藏名称或语句",
+      "expand-collapse": "展开/收起",
+      "expand": "展开",
+      "collapse": "收起",
+      "favorite-del-desc": "是否确认删除该条收藏语句？",
+      "input-query-statement": "请输入查询语句",
+      "query-statement-required": "查询语句不能为空",
+      "execute-query": "执行查询",
+      "execute-task": "执行任务",
+      "execute-ing": "执行中",
+      "clean": "清空",
+      "query-result-desc": "查询模式适合30秒内可返回结果的小规模分析；任务模式适合较长时间返回结果的大规模分析，任务详情可在任务管理中查看"
+    },
     "common": {
+      "in-edge": "入边",
+      "out-edge": "出边",
+      "add": "添加",
+      "id-strategy": "ID策略",
+      "id-value": "ID值",
+      "id-input-desc": "请输入ID值",
+      "please-input": "请输入",
+      "add-success": "添加成功",
+      "add-fail": "添加失败",
       "vertex-type": "顶点类型",
       "vertex-id": "顶点ID",
+      "add-vertex": "添加顶点",
+      "vertex-type-select-desc": "请选择顶点类型",
       "edge-type": "边类型",
+      "edge-type-select-desc": "请选择边类型",
       "edge-id": "边ID",
       "origin": "起点",
       "end": "终点",
```

Comment:
we generally use this term "source" and "target" for source vertex and target vertex, could you please rename the "origin" and "end" keys?

Reply:
fixed